src/main/java/org/eclipsefoundation/marketplace/helper/DateTimeHelper.java

```diff
@@ -0,0 +1,52 @@
+/* Copyright (c) 2019 Eclipse Foundation and others.
+ * This program and the accompanying materials are made available
+ * under the terms of the Eclipse Public License 2.0
+ * which is available at http://www.eclipse.org/legal/epl-v20.html,
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipsefoundation.marketplace.helper;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Central implementation for handling date time conversion in the service.
+ * Class needs to be instantiated as DateFormat objects are not safe by nature.
+ * 
+ * @author Martin Lowe
+ */
+public class DateTimeHelper {
```

Comment:
This helper class literally cries for a couple of unit tests

Reply:
You're right! While I was doing this, I ended up switching to use Java 8 time as its more explicit at handling timezones and offsets compared to previous Date implementation. The tests should cover most cases for the helper now.